pynio/instance.py

```diff
@@ -101,7 +104,7 @@ def _get_services(self):
         services = {}
         resp = self._get('services')
         for s in resp:
-            services[s] = Service(resp[s].get('name'),
+            services[s] = Service(resp[s].get('name', s),
```

Comment:
make sure the name is not None, also don't throw an error if a name isn't provided